IntelliTect.Multitool.Tests/RepositoryPaths.Tests.cs

```diff
@@ -10,4 +10,20 @@ public void GetDefaultRepoRoot_ReturnsRepoRootDirectory()
         // Makes the assumption that the repository directory for this solution is named the same as the solution
         Assert.EndsWith(nameof(Multitool), RepositoryPaths.GetDefaultRepoRoot());
     }
+
+    [Fact]
+    public void BuildVariables_BeingSetProperly()
+    {
+        Assert.Equal(3, RepositoryPaths.BuildVariables.Count);
+        
+        Assert.True(RepositoryPaths.BuildVariables.TryGetValue("BuildingForLiveUnitTesting", out _));
+        
+        Assert.True(RepositoryPaths.BuildVariables.TryGetValue("ProjectPath", out string? ProjectPath));
```

Comment:
nit: local named like a property. 
```suggestion
        Assert.True(RepositoryPaths.BuildVariables.TryGetValue("ProjectPath", out string? projectPath));
```